modules/ray_tracing/test/tests/trace_ray/edge4/tests

```diff
@@ -0,0 +1,22 @@
+[Tests]
+  design = 'TraceRay.md RayTracingStudy.md'
+  issues = '#16028'
+
+  [test]
+    type = 'CSVDiff'
+    input = '../lots.i'
+    csvdiff = 'lots_out.csv'
+    cli_args = 'Mesh/gmg/elem_type=edge4
+                Mesh/gmg/dim=1
+                RayBCs/active=kill_1d
+                UserObjects/lots/vertex_to_vertex=true
+                UserObjects/lots/centroid_to_vertex=true
+                UserObjects/lots/centroid_to_centroid=true
+                Outputs/file_base=lots_out'
+    allow_test_objects = true
+    abs_zero = 1e-8
+    method = '!dbg'
+    skip = 'edge4 issue'
```

Comment:
is this valid? Dont you need a number

Reply:
It's temporary. will be fixed in this PR